lumicks/pylake/scan.py

```diff
@@ -14,16 +15,19 @@ class Scan(Kymo):
     file : lumicks.pylake.File
         The parent file. Used to loop up channel data
     """
-    def __init__(self, h5py_dset, file):
-        super().__init__(h5py_dset, file)
-        self._num_frames = self.json["scan count"]
-        if len(self.json["scan volume"]["scan axes"]) > 2:
+    def __init__(self, name, file, start, stop, metadata):
+        super().__init__( name, file, start, stop, metadata)
+        self._num_frames = self.metadata["scan count"]
+        if len(self.metadata["scan volume"]["scan axes"]) > 2:
```

Comment:
Renaming `scan.json` to `scan.metadata` is a breaking API change since this is not a private attribute (doesn't start with an underscore). I'd suggest either reverting this or adding property alias with a deprecation warning for a transition period.

Reply:
Alright. Reverted.

Reply:
My bad, I suggested that. It's hard to remember that parameter names are part of the API in python... :sweat_smile: